src/solver/conv_asm_implicit_gemm_gtc_fwd.cpp

```diff
@@ -1536,6 +1531,11 @@ bool ConvAsmImplicitGemmGTCDynamicFwdXdlops::IsApplicable(const ConvolutionConte
         return false;
     }
 
+#if WORKAROUND_SWDEV_306318
+    if((ctx.kernel_size_h == 1) && (ctx.kernel_size_w == 1) && (ctx.n_inputs % 8 != 0))
+        return false;
```

Comment:
It is recommended to insert
```C++
if(!miopen::IsEnabled(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_FWD_GTC_XDLOPS{}))
```
just above `return false`. This would allow explicit enabling the solver for `1x1 C%8=0` configs for triaging, without rebuilding the library.

Reply:
@atamazov Do you mean with the current commits,  `1x1 c%8 !=0` configs could not be tested, even after the kernels being improved?   and by inserting 
`if(!miopen::IsEnabled(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_FWD_GTC_XDLOPS{}))` 
 we are able to test `1x1 c%8 !=0` by using 
`export MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_FWD_GTC_XDLOPS=1` ? 

So my understand is that the env var  `MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_FWD_GTC_XDLOPS` should have three states:
1) Disabled ( defined from env and have false value) 
2) Enabled ( defined from env and have true value)
3) Neither Disabled nor Enabled ( not defined from env) 

right?

Reply:
@qianfengz Yes, and in the latter case both IsEnabled and IsDisabled return false. This is useful in the cases like this one.

Reply:
> It is recommended to insert
> 
> ```c++
> if(!miopen::IsEnabled(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_FWD_GTC_XDLOPS{}))
> ```
> 
> just above `return false`. This would allow explicit enabling the solver for `1x1 C%8 != 0` configs for triaging, without rebuilding the library.

@carlushuang  please take a look at the atamazov recommendation, so we will approve after your update